admin-site/src/App.svelte

```diff
@@ -0,0 +1,421 @@
+						<span class="fas fa-angle-left"></span>
+						Previous
+					</button>
+				</div>
+				<div class="col-md-6">
+					<button type="button" class="btn btn-info btn-block" value="next" on:click={changeCueState}>
+						Next
+						<span class="fas fa-angle-right"></span>
+					</button>
+				</div>
+			</div> 
+
+			<div class="row">
+				<div class="col-md-12">
+					<Slider/>
+				</div>
+			</div>
+	</div>
+</div>
+
+<div id = "QRcode">
+	<div class="modal-dialog modal-lg">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="Close" value= "QRcode" on:click={backToOccasionList}>
+					<span aria-hidden="true">&times;</span>
+				</button>
+			</div>
+			<div class="modal-body">
+				<div class="container-fluid">
+					<div class="row">
+						<div class="col-md-12 text-center">
+							<img src="QrCodes/Event1.png " class="img-fluid" alt="QR Code for">
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div id = "closeEvent">
+	<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+        			<h5 class="modal-title">{focusedEvent.label} - {formattedStartTime}.</h5>
+				</div>
+			</div>
+			<div class="row">
+				<button type='button' class="btn btn-primary" value="closeEvent" on:click={backToOccasionList}>Back To Occasions List</button>
+			
+			</div>
+			<div class="row">
+				<div class="col-md-12">
+					<label for="OccasionDetails"><h4>Occasion Details</h4></label>
+					<ul id="OccasionDetails">
+						<li>Start Date : {formattedStartTime}</li>
+						<li>End Date : {formattedEndTime}</li>
+						<li>Location Label: {focusedOccasion.locationLabel} </li>
+						<li>Location Address: {focusedOccasion.locationAddress} </li>
+						<li>Location City: {focusedOccasion.locationCity} </li>
+					</ul> 	
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-danger btn-block" on:click={confirmOccasionDelete}>Delete Occasion</button>
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-success btn-block" on:click={openOccasionButton}>Open Occasion</button>
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-primary btn-block" value ="closeEvent" on:click={showQR}>Get QR Code</button>
+				</div>
+			</div>
+	</div>
+</div>
+
+<!-- //modals show/hide not working, so removed below so they can act as a static document 
+<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true"></div> -->
+<div id ="confirmDelete">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="deleteOccasionConfirmation">Confirmation</h5>
```

Comment:
Change text to the main action: "Delete Occasion"

Reply:
Change applied